roles/ceph-mon/templates/ceph-mon.service.j2

```diff
@@ -23,8 +23,13 @@ ExecStart=/usr/bin/docker run --rm --name ceph-mon-%i --net=host \
    --net=host \
    {% endif -%}
    -e CEPH_DAEMON=MON \
-   -e MON_IP={{ hostvars[inventory_hostname]['ansible_default_' + ip_version]['address'] }} \
    -e CEPH_PUBLIC_NETWORK={{ ceph_mon_docker_subnet }} \
+   {% if ip_version == 'ipv4' %}
+   -e MON_IP={{ hostvars[inventory_hostname]['ansible_' + ceph_mon_docker_interface][ip_version]['address'] }} \
```

Comment:
if I'm not mistaking the default value for IP_VERSION is '4' in ceph-docker.
You are suggesting we should specify it in here anyway, right?